docs/using-the-python-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md

```diff
@@ -104,6 +104,7 @@ conn = AwsWrapperConnection.connect(psycopg.Connection.connect, **params)
 | `random`                        | The random strategy is the default connection strategy. When switching to a reader connection, the reader instance will be chosen randomly from the available database instances.                                                                                                                                                                                                                                                                                                                                                                                 | N/A                                                                                                        |
 | `least_connections`             | The least connections strategy will select reader instances based on which database instance has the least number of currently active connections. Note that this strategy is only available when internal connection pools are enabled - if you set the connection property without enabling internal pools, an exception will be thrown.                                                                                                                                                                                                                        | N/A                                                                                                        |
 | `round_robin`                   | The round robin strategy will select a reader instance by taking turns with all available database instances in a cycle. A slight addition to the round robin strategy is the weighted round robin strategy, where more connections will be passed to reader instances based on user specified connection properties.                                                                                                                                                                                                                                             | See the rows `round_robin_host_weight_pairs` and `round_robin_default_weight` for configuration parameters |
+| `weighted_random`               | The weighted random strategy will be chosen randomly from the available database instances. A slight addition to the random strategy is the weighted random strategy, where more connections will be passed to reader instances based on user specified connection properties.                                                                                                                                                                                                                                                                                    | See the rows `round_robin_host_weight_pairs` and `round_robin_default_weight` for configuration parameters |
```

Comment:
Incorrect reference to round robin.
We should put this row after the round robin parameters for readability